ergo-lib/Cargo.toml

```diff
@@ -33,6 +33,7 @@ proptest-derive = {version = "0.3.0", optional = true }
 sha2 = { version = "0.9.8" }
 hmac = { version = "0.11.0" }
 pbkdf2 = "0.8"
+bs58 = "0.4.0"
```

Comment:
Are you sure it's needed? I did not find any usage in the code.

Reply:
At the moment only used for comparing against node test vectors: https://github.com/ergoplatform/sigma-rust/pull/498/files#diff-214500aee5bfd9edaf034a03b5e268694ba68ef0bd0b3c79107ac39222b273c3R275

Reply:
Oh, I missed it. I was running grep on github's PR webpage. :)
Can you move it to dev-dependencies then, or do you think we're going to need it here soon anyway?

Reply:
Moved to `dev-deps`, easy to move back to `deps` if it's ever needed 👍